src/_shared/utils/applyApTitleCase.ts

```diff
@@ -1,10 +1,24 @@
 export const STOP_WORDS =
-  'a an and at but by for in nor of on or so the to up yet';
+  'a an and at but by for in nor of on or the to up yet';
 
-export const SEPARATORS = /(\s+|[-‑–—,:;!?()“”"])/;
+export const SEPARATORS = /(:\s*|\s+|[-‑–—,:;!?()“”'‘"])/; // Include curly quotes as separators
```

Comment:
for readability for future us, would it be feasible to add a comment here translating the different pieces of this regex?

i'm specifically curious about this part: `:\s*|\s+` i am not a regex wizard, but why is the `\s+` necessary? doesn't `\s*` cover that case?

Reply:
`:\s*` matches the `:` char & 0+ white spaces following the colon.
`\s+` matches only 1+ white spaces. 
So if we only had `:\s*` in the regex, it would only capture the `:` and any (or 0) white spaces trailing after. But we also want to capture just white spaces, and `\s+` allows you to do that.

Reply:
oh, i see - this is why i'm not a regex wizard. the `:\s*` and `\s+` are fully separate. i think i was reading it as `(:(\s*|\s+)|[-‑–—,:;!?()“”'‘"])/`

Reply:
i am no regex wizard either, https://regex101.com/ has my back whenever I need to test a regex 😅